docs/framework/winforms/controls/how-to-set-tooltips-for-controls-on-a-windows-form-at-design-time.md

```diff
@@ -44,7 +44,32 @@ You can set a <xref:System.Windows.Forms.ToolTip> string in code or in the Windo
 2.  Select the control that will display the ToolTip, or add it to the form.  
   
 3.  In the **Properties** window, set the **ToolTip on ToolTip1** value to an appropriate string of text.  
+
+### To remove a ToolTip programmatically  
   
+1.  Use the <xref:System.Windows.Forms.ToolTip.SetToolTip%2A> method of the <xref:System.Windows.Forms.ToolTip> component.  
+  
+    ```vb  
+    ' In this example, Button1 is the control displaying the ToolTip.  
+    ToolTip1.SetToolTip(Button1, Nothing)  
+    ```  
+  
+    ```csharp  
+    // In this example, button1 is the control displaying the ToolTip.  
+    toolTip1.SetToolTip(button1, null);  
+    ```  
+  
+    ```cpp  
+    // In this example, button1 is the control displaying the ToolTip.  
+    toolTip1->SetToolTip(button1, NULL);  
+    ```  
+  
+### To remove a ToolTip in the designer  
+  
+1.  Select the control that is displaying the ToolTip.  
+  
+2.  In the **Properties** window, set the **ToolTip on ToolTip1** value to an empty string.  
```

Comment:
This doesn't remove the tooltip -- it sets the tooltip text for the control to an empty string, which is displayed as "". To remove the tooltip, delete the existing text; that's the equivalent of setting it to null programmatically.